src/Microsoft.AspNet.Authorization/DefaultAuthorizationService.cs

```diff
@@ -33,7 +50,17 @@ public DefaultAuthorizationService(IOptions<AuthorizationOptions> options, IEnum
             {
                 await handler.HandleAsync(authContext);
             }
-            return authContext.HasSucceeded;
+
+            if (authContext.HasSucceeded)
+            {
+                _logger.LogInformation(0, $"Authorization was successful for user: {user?.Identity?.Name}.");
+                return true;
+            }
+            else
+            {
+                _logger.LogInformation(0, $"Authorization failed for user: {user?.Identity?.Name}.");
```

Comment:
Grrr, copy paste fail, yeah event ids, manually assigning them for now, 0 and 1 :)